site2/docs/reference-configuration.md

```diff
@@ -215,7 +215,7 @@ internalListenerName|Specify the internal listener name for the broker.<br><br>*
 |brokerClientTlsTrustStorePassword| TLS TrustStore password for internal client, used by the internal client to authenticate with Pulsar brokers ||
 |brokerClientTlsCiphers| Specify the tls cipher the internal client will use to negotiate during TLS Handshake. (a comma-separated list of ciphers) e.g.  [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]||
 |brokerClientTlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS handshake. (a comma-separated list of protocol names). e.g.  [TLSv1.2, TLSv1.1, TLSv1] ||
-|ttlDurationDefaultInSeconds|  The default ttl for namespaces if ttl is not configured at namespace policies.  |0|
+|ttlDurationDefaultInSeconds| The default TTL for namespaces if TTL is not configured at namespace policies. TTL(time to live) is disabled by default and the value is set to `0`. |0|
```

Comment:
@Jennifer88huang should we call out that setting it to `0` is disabling TTL and the default value is `0`.

Also, shall we update the documentation of other releases?

Reply:
@sijie you're so fast, I've just updated it to be more clear. Please check the [latest version above](https://github.com/apache/pulsar/pull/9395/commits/4e04618586577841c1dd258b35d0ffa1182712cf)

Reply:
Yes, we will apply changes to other versions if it should. 
Usually, we update in the master and finalize the change first, and then apply the final version content to other versions. Thank you for your reminder.